harvest/wizard/robinhood.py

```diff
@@ -0,0 +1,91 @@
+# Builtins
+from getpass import getpass
+
+# External libraries
+import pyotp
+
+# Submodule imports
+from harvest.wizard._base import Wizard
+
+class RobinhoodWizard(Wizard):
```

Comment:
The interface looks awesome :). Though I was thinking it might be easier if instead we instantiate the Wizard class in RobinhoodBroker, and call the interface like `wizard.get_bool()`. That way we don't need to make a class for every broker.

Reply:
For now I think we can merge though, I'll work on what I just mentioned + some little tweaks tonight.